noxfile.py

```diff
@@ -52,6 +53,12 @@ def install_dev_env(session: Session):
     session.run(str(install_script))
 
 
+@nox.session(python=False)
+def amalgate_lua_scripts(session: Session):
+    script = ROOT_DIR / "exasol_advanced_analytics_framework" / "deployment" / "regenerate_scripts.py"
+    _run_in_dev_env_poetry_call(session, "python", str(script))
```

Comment:
what is this for, could a nox based venv be a solution here?

Reply:
We need to consider the Lua installation and conda being involved. 
Potentially, this might be possible. 
Not sure. 

However, these parts are currently unchanged and only wrapped into nox tasks in some crucial places. 
In order to not merge too many changes into a single PR I propose to create a separate ticket. 
Could you agree to that?

Reply:
@Nicoretti this is unfortunately a nasty combination out of a conda env, with luarocks env and a poetry env, if you have an idea how we get this done with nox I am happy to hear